frontend/app/src/locales/en.json

```
@@ -188,6 +188,7 @@
   },
   "backup_manager": {
     "backup_button": "Create new Backup",
+    "delete_selected": "Delete selected Backup",
```

```suggestion
    "delete_selected": "Delete selected Backups",
```
Since you can have multiple backups selected let's keep it plural